engine/collection/compliance/core.go

```diff
@@ -83,7 +83,7 @@ func (c *Core) OnBlockProposal(originID flow.Identifier, proposal *messages.Clus
 		Hex("payload_hash", header.PayloadHash[:]).
 		Time("timestamp", header.Timestamp).
 		Hex("proposer", header.ProposerID[:]).
-		Int("num_signers", len(header.ParentVoterIDs)).
+		// Int("num_signers", len(header.ParentVoterIDs)).
```

Comment:
I commented this out because I haven't thought about what to replace with, but also don't want to forget about it.

Now the header only has ParentVoterIndices, I don't want to take extra query to parse the actual signers. Maybe we could simply log the entire `ParentVoterIndices` field?

Reply:
Addressed [here](https://github.com/onflow/flow-go/pull/2140/files#diff-6491debb977ace8c99ddb1edbd37fca0420cc659243a52ebb03505ba7efe78fdR86)